lib/THNN/generic/VolumetricConvolutionMM.c

```diff
@@ -2,6 +2,20 @@
 #define TH_GENERIC_FILE "generic/VolumetricConvolutionMM.c"
 #else
 
+static int THNN_(view_weight)(THTensor **_weight)
+{
+  THTensor *weight = *_weight;
+  THArgCheck(weight->nDimension == 2 || weight->nDimension == 5, 4,
+          "weight tensor should be 2D or 5D - got %dD", weight->nDimension);
+  if (weight->nDimension == 5) {
+    long s1 = weight->size[0];
+    long s2 = weight->size[1] * weight->size[2] * weight->size[3] * weight->size[4];
+    *_weight = THTensor_(newWithStorage2d)(weight->storage, 0, s1, -1, s2, -1);
```

Comment:
Need to add `storageOffset` here, as was included in
https://github.com/torch/nn/blob/master/lib/THNN/generic/SpatialConvolutionMM.c#L104